examples/reference/widgets/ChatInterface.ipynb

```diff
@@ -160,7 +162,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Widgets other than `TextInput` will require the user to manually click the `Send` button, unless the type is specified in `auto_send_types`."
+    "Widgets other than `TextInput` and `TextAreaInput` will require the user to manually click the `Send` button, unless the type is specified in `auto_send_types`."
```

Comment:
I don't know if it works. But I believe it should be easy to replace with the `TextAreaInput`. Maybe also with other text inputs like `TextEditor` and `CodeEditor`.